annubes/task.py

```diff
@@ -165,42 +162,32 @@ def generate_trials(
         trials["outputs"] = self._outputs
         return trials
 
-    def _rescale(
+    def _minmaxscaler(
         self,
-        input_: float,
-        coeff: float,
-        min_intensity: float | None = None,
-        max_intensity: float | None = None,
+        input_: NDArray[np.float32],
+        input_range: tuple[float, float] = (0, 1),
```

Comment:
I think `input_range` is a bit confusing nomenclature. I believe you mean it's the range of the input data of `Task` after rescaling. But it sounds like it's the range of the input to this method.
Perhaps we can call it `rescale_range` instead?
```suggestion
        rescale_range: tuple[float, float] = (0, 1),
```